src/components/modules/dropdown/select.jsx

```diff
@@ -427,7 +426,7 @@ export default class Select extends React.Component {
         let newChildren = [];
         React.Children.forEach(this.props.children, child => {
             // only process option like childs
-            if (child.props.value) {
+            if (typeof child.props.value !== 'undefined') {
```

Comment:
I am checked and don't see problems. Everything works as before, but now the value may be all but `
undefined
`

update example https://github.com/hallister/semantic-react/pull/84/commits/4590d1d00c9f70b5d4233fd3b07f17ee262f97a2